app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -193,6 +193,12 @@ def datacenters(options = {})
       client.datacenters(options).map { |dc| [dc[:name], dc[:id]] }
     end
 
+    def get_datacenter_uuid(name)
+      datacenter_uuid = datacenters.select{|dc| dc[0] == name}
```

Comment:
What happens if a user passes UUID as `name`?

Reply:
You are right, the code was supposed to return if the datacenter is a UUID, I must have deleted it by accident at some point. That is what was expected and why the `unless`exists: https://github.com/theforeman/foreman/pull/5157/files#diff-cf2965b610f2fa0c776c7129f985f26dR63

Reply:
@ShimShtein I added: https://github.com/theforeman/foreman/pull/5157/files#diff-cf2965b610f2fa0c776c7129f985f26dR60

Reply:
@orrabin, That is a good addition, although in case of a uuid, `@compute_resource.datacenter` will not be set.